pdns/recursordist/docs/manpages/pdns_recursor.1.rst

```diff
@@ -76,6 +76,8 @@ at `<https://doc.powerdns.com/>`
     Load root hints from this *filename*
 --local-address=<address>
     Listen on *address*, separated by spaces or commas.
+    Addresses specified can include port numbers; any which do not
+    incude port numbers will listen on *--local-port*.
```

Comment:
[check spelling](https://github.com/jsoref/pdns/commit/91c2e7e361bf63480564bff8e0ae783d28800134#commitcomment-39229194) will complain about this.
```suggestion
    include port numbers will listen on *--local-port*.
```

Reply:
Done.